play-media-native/screens/EditAthleteDetails.tsx

```diff
@@ -0,0 +1,223 @@
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { useEffect, useState } from "react";
+import {
+  SafeAreaView,
+  ScrollView,
+  TouchableOpacity,
+  View,
+  StyleSheet,
+  TextInput,
+} from "react-native";
+import { Button, Text } from "react-native-paper";
+import { useQuery } from "react-query";
+import { getAthleteById } from "../api/queries/getAthletes";
+import { LoadingScreen } from "../features/LoadingScreen/LoadingScreen";
+import { getAccentColor } from "../helpers/colorHelper";
+import { getDate, getYear } from "../helpers/dateHelper";
+import { theme } from "../theme/theme";
+import DateTimePicker from "@react-native-community/datetimepicker";
+import { Athlete } from "../interfaces/athlete";
+import { AthleteImages } from "../features/Screens/AthleteImages";
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: theme.colors.black.darkest,
+    paddingTop: theme.spacing.md,
+    paddingHorizontal: theme.spacing.sm,
+  },
+  button: {
+    position: "absolute",
+    right: -theme.spacing.sm,
+    top: -theme.spacing.xs,
+  },
+  label: {
+    color: theme.colors.white.DEFAULT,
+    marginBottom: theme.spacing.xxs,
+  },
+  item: {
+    marginBottom: theme.spacing.xs,
+    fontFamily: theme.fontFamily.bold,
```

Comment:
Shouldn't be bold. See: https://dev.azure.com/sitecoredst/Demo/_boards/board/t/Demo%20Solutions%20Team/Stories/?workitem=8450